rocAL/rocAL_pybind/examples/rocAL_api_pytorch_classification_reader.py

```diff
@@ -18,189 +18,81 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
+import random
 from amd.rocal.plugin.pytorch import ROCALClassificationIterator
 from amd.rocal.pipeline import Pipeline
 import amd.rocal.fn as fn
 import amd.rocal.types as types
-from parse_config import parse_args
+import numpy as np
+import sys
 import os
 
-def draw_patches(img, idx, device):
-    #image is expected as a tensor, bboxes as numpy
+def draw_patches(img, idx, layout="nchw", dtype="fp32", device="cpu"):
+    #image is expected as a tensor
     import cv2
-    args = parse_args()
     if device == "cpu":
-            image = img.detach().numpy()
+        image = img.detach().numpy()
     else:
         image = img.cpu().numpy()
-    if not args.NHWC:
+    if layout == "nchw":
         image = image.transpose([1, 2, 0])
-    if args.fp16:
-        image = (image).astype('uint8')
+    if dtype == "fp16":
```

Comment:
the code need to get correct image output from the pipeline

Reply:
Will make these args as user given args in future PRs. This is just a basic example provided to showcase changes